project-base/app/tests/App/Functional/Component/Redis/RedisVersionsFacadeTest.php

```diff
@@ -15,7 +15,9 @@ public function testCleanOldCache(): void
         $oldVersionKey = '20190308130158.test';
         $notVersionKey = '2019030813015.test';
 
-        $redisClient = self::getContainer()->get('snc_redis.test');
+        /** @var \Redis $redisClient */
+        $redisClient = self::getContainer()->get('snc_redis.global');
```

Comment:
Why is this change necessary?

Reply:
The test for redis version facade was not testing properly. 
The old versions were not deleted, yet the test was still green. With use of the same redis client as is used to delete old versions, we can be sure the deleting old versions work properly